### PR TITLE
Cooking Bugfixes Round 1

### PR DESCRIPTION
**Pull request by MarinaGryphon**
* Fixes a number of cooking bugs. Detailed changelog soon.

### CODE REVIEW

**Comment by MarinaGryphon** [comment]
Fixes #9897, #9883

**Comment by MarinaGryphon** [comment]
Also maybe fixes #9896

**Comment by MarinaGryphon** [comment]
!bugfix

File: code/modules/cooking/machinery/cooking_machines/_cooker_output.dm [hunk 1]
```diff
@@ -22,37 +22,51 @@
...
 
-/obj/item/reagent_containers/food/snacks/variable/update_icon()
+/obj/item/reagent_containers/food/snacks/variable/on_reagent_change()
+	return
+
+/obj/item/reagent_containers/food/snacks/variable/proc/update_prefix()
+	switch(scale)
+		if (0 to 0.8)
+			prefix = "small"
+		if (0.8 to 1.2)
```
Comment (alsoandanswer) [inline code/modules/cooking/machinery/cooking_machines/_cooker_output.dm:32]: there's no intermediate/medium size? 

should remove extra large and just bump everything up

Comment (MarinaGryphon) [reply code/modules/cooking/machinery/cooking_machines/_cooker_output.dm:32]: Bugfix PR.

**Review by Geevies — APPROVED** [review]

**Review by skull132 — APPROVED** [review]

**Pull request merged**